internal/store/engine_state.go

```diff
@@ -54,6 +54,9 @@ type EngineState struct {
 	GlobalYAMLState *YAMLManifestState
 
 	TiltfilePath string
+
+	// InitManifests is the list of manifest names that we were told to init from the CLI.
+	InitManifests []string
```

Comment:
Shouldn't this be a list of ManifestNames? This just creates extra work to convert to strings

Reply:
Yeah agreed